catalog/imaging/imaging-plugin-nitf/src/main/java/org/codice/alliance/plugin/nitf/NitfPostIngestPlugin.java

```diff
@@ -115,81 +123,133 @@
 
   private boolean storeOriginalImage = true;
 
+  private CatalogFramework catalogFramework;
+
   static {
     IIORegistry.getDefaultInstance().registerServiceProvider(new J2KImageReaderSpi());
   }
 
   private double maxSideLength = DEFAULT_MAX_SIDE_LENGTH;
 
   @Override
-  public CreateStorageRequest process(CreateStorageRequest createStorageRequest)
-      throws PluginExecutionException {
-    if (createStorageRequest == null) {
-      throw new PluginExecutionException(
-          "process(): argument 'createStorageRequest' may not be null.");
+  public CreateResponse process(CreateResponse createResponse) throws PluginExecutionException {
+    if (createResponse == null) {
+      throw new PluginExecutionException("process(): argument 'createResponse' may not be null.");
```

Comment:
Do we need to check if the createResponse is null? In the API we do not state it as nullable. If we can't guarantee it should be null in practice we should make a note to go over it with the core-apis team.

Reply:
It should never be null but some plugins check for it anyway. This one already did so I just kept it.